client/login/index.web.js

```diff
@@ -73,7 +73,6 @@ export default ( router ) => {
 
 		router(
 			[ `/log-in/link/${ lang }`, `/log-in/jetpack/link/${ lang }`, `/log-in/new/link/${ lang }` ],
```

Comment:
Could the other magic login similar paths gain from a similar treatment (`/log-in/link/use` and `/log-in/jetpack/link/use`)? What about `/log-in/qr`?

I'd say any of them that supports the `redirect_to` parameter could benefit from that.

And if we end up using that in multiple places, perhaps we could abstract it to its own controller function?

WDYT?

Reply:
Thank you @tyxla! I don't have the time Today to be responsible of those changes but will add it to my TODO list.

Reply:
Sounds good! Not a blocker in any way.